src/Tizen.Network.Tethering/Tizen.Network.Tethering/TetheringExtClient.cs

```diff
@@ -0,0 +1,130 @@
+using System;
+
+namespace Tizen.Network.Tethering
+{
+    /// <summary>
+    /// This class contains information related to the connected/disconnected client.
+    /// </summary>
+    /// <since_tizen> 13 </since_tizen>
+    public class TetheringExtClient : IDisposable
```

Comment:
we have namespaces, prefixing is not needed, consider name like simple "Client" or "ClientInformation" (if the class is not expected to offer any operations)